tests/interfaces/test_gen_image_similarity_blackbox_sal.py

```diff
@@ -0,0 +1,142 @@
+import unittest.mock as mock
+import pytest
+import numpy as np
+
+from smqtk_descriptors.interfaces.image_descriptor_generator import ImageDescriptorGenerator
+
+from xaitk_saliency import GenerateImageSimilarityBlackboxSaliency
+from xaitk_saliency.exceptions import ShapeMismatchError, UnexpectedDimensionsError
+
+
+def test_generate_checks_success() -> None:
+    """
+    Test successful passage through the wrapper method.
+    """
+    m_impl = mock.Mock(spec=GenerateImageSimilarityBlackboxSaliency)
+
+    # test images
+    test_ref_image = np.empty((50, 50))
+    test_query_image = np.empty((256, 256, 7))
+
+    # mock _generate result with matching height and width to query image
+    m_impl._generate.return_value = np.empty((256, 256))
+
+    # mock image descriptor generator
+    m_desc_generator = mock.Mock(spec=ImageDescriptorGenerator)
+
+    GenerateImageSimilarityBlackboxSaliency.generate(
```

Comment:
We should additionally test here that:
1) the _generate method was called with the expected inputs
1) the result returned here is the same as what the generator output